### PR TITLE
fix(duckduckgo): partially unthemed popup

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
as said in the title

image
image

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/duckduckgo/catppuccin.user.css [hunk 1]
```diff
@@ -297,6 +297,12 @@
       background: linear-gradient(rgba(40, 40, 40, 0), @surface0);
     }
 
+    /* popup */
+    .atb-new .badge-link__checkbox.badge-link__checkbox_checked {
+        background: @mantle;
+        color: @text;
+    }
```
Comment (uncenter) [inline styles/duckduckgo/catppuccin.user.css:304]: Are you sure there aren't variables we can use here? I'm seeing other already themed variables with `atb` in them which makes me think we could use vars here.

Comment (somerand0mcat) [reply styles/duckduckgo/catppuccin.user.css:304]: i didnt check to see if there was any variables which we can use, maybe you can theme these ``atb`` variables and check to see if they actually theme?

Comment (uncenter) [reply styles/duckduckgo/catppuccin.user.css:304]: Yes... I'm asking you to do that. You should *always* be checking for variables.

Comment (somerand0mcat) [reply styles/duckduckgo/catppuccin.user.css:304]: i checked the variables and theres no way we can use them here so this codes good

**Comment by somerand0mcat** [comment]
merged with #1077

**Pull request closed without merging**